rasa/shared/nlu/training_data/formats/rasa_yaml.py

```diff
@@ -23,6 +23,7 @@
 
 from rasa.shared.nlu.training_data.training_data import TrainingData
 from rasa.shared.nlu.training_data.message import Message
+from rasa.shared.nlu.training_data.util import has_string_escape_chars
```

Comment:
minor: We have a Python Code convention to not import functions from modules but rather their modules (see [here](https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e#900c90130b4e4fe6b688f30c3826e764))